addons/web/static/src/legacy/scss/bootstrap_overridden.scss

```
@@ -273,3 +262,13 @@ $badge-padding-x: 0.4em !default;
 
 // Placeholder color 
 $input-placeholder-color: $gray-300 !default;
+
+// Card
+
+$card-spacer-y:                     $spacer;
```

Add "BS default" notes accordingly.
This files is normally for overrides, but its my understanding that you have to re-define some variables to use them later on
```suggestion
$card-spacer-y:                     $spacer; // BS Default
```